src/scancode/cli.py

```diff
@@ -518,7 +541,7 @@ def _scanit(paths, scanners, scans_cache_class, diag,
     return success, rel_path
 
 
-def resource_paths(base_path):
+def resource_paths(base_path, user_ignores):
```

Comment:
Could you add a few few unit tests for this function, especially now that there would be ignores support?